messaging/2.0/_index.md

```diff
@@ -1,11 +1,18 @@
 ---
 title: "Jakarta Messaging 2.0"
 date: 2019-07-19T15:57:57-0700
-summary: "Draft for Jakarta EE 8"
+summary: "Final release for Jakarta EE 8"
 ---
 
 Jakarta Messaging describes a means for Java applications to create, send, and receive messages via loosely coupled, reliable asynchronous communication services.
 
+* [Jakarta Messaging 2.0 Specification Document](./messaging-spec-2.0.pdf) (PDF)
+* [Jakarta Messaging 2.0 Specification Document](./messaging-spec-2.0.html) (HTML)
+* [Jakarta Messaging 2.0 Javadoc](./apidocs)
+* [Jakarta Messaging 2.0 TCK](https://download.eclipse.org/jakartaee/messaging/2.0/jakarta-messaging-tck-2.0.0.zip)
```

Comment:
The zip file is named eclipse-messaging-tck-2.0.0.zip.

Reply:
There was discussion in one of the spec meetings that anything named "eclipse-<foo>-tck" would get renamed to "jakarta-<foo>-tck" during the signing.  Specifically, that "Eclipse Messaging" is not acceptable brand usage and the lowest impact way to fix it is on the final copy vs making everyone go update their work-in-progress.  There would be a tweak of the _index.md to adjust, which didn't concern us as we don't expect 100% of all links to be perfect.  We can definitely recap on the Spec call tomorrow.

Reply:
I agree that we discussed this.  And, we will be changing these names to jakarta- before we ship.  But, since this will be a change through out all of the TCK references, wouldn't it be easier to follow suit for now?  It would also make it easier for reviewing as we continue this process.  Thanks.

Reply:
> There was discussion in one of the spec meetings that anything named "eclipse--tck" would get renamed to "jakarta--tck" during the signing.

I don't remember that at all.  Can you find that in the meeting minutes?

What I do remember is you saying that your signing script would explicitly **not** change the name of the zip file.